gems/pending/VMwareWebService/MiqVimInventory.rb

```diff
@@ -2176,9 +2174,7 @@ def getSelSpec(selSpec)
       ss = getSelector(ss)
       raise "getSelSpec: selector #{selSpec} not found" if ss.nil?
     end
-    ss = [] if ss.nil?
-    ss = [ss] unless ss.kind_of?(Array)
-    ss
+    Array(ss)
```

Comment:
This is actually different than previously depending on what ss is.  If it's a Hash:

```ruby
ss = {:a => 1}
ss = [] if ss.nil?
ss = [ss] unless ss.kind_of?(Array)
ss
# => {:a => 1}
```

```ruby
ss = {:a => 1}
Array(ss)
# => [[:a, 1]]